satpy/etc/readers/modis_l2.yaml

```diff
@@ -143,12 +148,75 @@ datasets:
     file_type: mod35_hdf
     coordinates: [longitude, latitude]
 
+##########################
+#Datasets in file mod05_l2
+##########################
+  scan_start_time:
+    name: scan_start_time
+    long_name: TAI time at start of scan replicated across the swath
+    units: seconds since 1993-1-1 00:00:00.0 0
+    file_type: mod05_hdf
+    coordinates: [longitude, latitude]
+    resolution:
+      5000:
+        file_key: Scan_Start_Time
+
+  sensor_zenith:
+    name: sensor_zenith
+    long_name: Sensor Zenith Angle, Cell to Sensor
```

Comment:
For consistency with the rest of Satpy, what do people think about naming this `satellite_zenith_angle`?

Reply:
Yes, good idea.

Reply:
done